packages/shopware-6-client/src/interfaces/response/Error.ts

```diff
@@ -0,0 +1,19 @@
+interface Error {
+  status: string;
+  code: string;
+  title: string;
+  detail: string;
+  meta: any;
+  trace?: Array<{
```

Comment:
could you move it to separate interface?
```suggestion
  trace?: ErrorTrace[];
```